firefox_decrypt.py

```diff
@@ -430,6 +431,14 @@ def output_line(line):
         LOG.info("Decrypting credentials")
         to_export = {}
 
+        if format == 'csv':
+            csv_writer = csv.DictWriter(
+                sys.stdout, fieldnames={'host', 'user', 'password'},
+                delimiter=';', quotechar='|', quoting=csv.QUOTE_ALL
+            )
```

Comment:
I think `"` is a more suitable quote-char. Maybe expose this to the user somehow?